nixos/modules/services/desktops/pipewire/pipewire-media-session.nix

```diff
@@ -38,8 +38,8 @@ in {
     services.pipewire.media-session = {
       enable = mkOption {
         type = types.bool;
-        default = config.services.pipewire.enable;
-        defaultText = literalExpression "config.services.pipewire.enable";
+        default = false;
```

Comment:
Should this generate a deprecation warning if enabled?

Reply:
I'm honestly not sure. Maybe once Wireplumber gets some more testing?